runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -3735,6 +3736,15 @@ struct amRequest_AMO_t {
   void* result;                 // result address on initiator's node
 };
 
+struct amRequest_FAMO_result_t {
+  struct amRequest_base_t b;
+  int8_t size;                  // object size (bytes)
+  chpl_amo_datum_t object;      // the object
+  enum fi_datatype ofiType;     // ofi object type
+  void* obj;                    // object address on target node
```

Comment:
Re: the comment: agreed that we're effectively PUTting something to a remote node, but in the larger sense because this is a response (to the original fetching atomic request) I think I would refer to this as the "initiating node" rather than the "target node" or, if we want to be more specific, the "initiating node of the original fetching atomic op".